sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -642,9 +642,9 @@ def list_blobs(self, name_starts_with=None, include=None, **kwargs):
         :param str name_starts_with:
             Filters the results to return only blobs whose names
             begin with the specified prefix.
-        :param list[str] include:
+        :param list[str] or str include:
```

Comment:
Could you update the type annotation too?

Reply:
what do you mean?